forge-gui/src/main/java/forge/gamemodes/limited/LimitedPoolType.java

```diff
@@ -5,16 +5,45 @@
 public enum LimitedPoolType {
     Full(Localizer.getInstance().getMessage("lblLimitedPoolFull")),
     Block(Localizer.getInstance().getMessage("lblLimitedBlock")),
-    Prerelease(Localizer.getInstance().getMessage("lblLimitedPrerelease")),
+    Prerelease(Localizer.getInstance().getMessage("lblLimitedPrerelease"), false),
     FantasyBlock(Localizer.getInstance().getMessage("lblLimitedFantasy")),
     Custom(Localizer.getInstance().getMessage("lblLimitedCustom")),
     Chaos(Localizer.getInstance().getMessage("lblLimitedChaos"));
-    
+
     private final String displayName;
+    private final boolean draftable;
+
     LimitedPoolType(String name) {
+        this(name, true);
+    }
+
+    LimitedPoolType(String name, boolean draftable) {
+        this.draftable = draftable;
         displayName = name;
     }
 
+    public static LimitedPoolType[] values(boolean draftable) {
+        if (!draftable) {
+            return values();
+        }
+
+        int n = 0;
+        for (LimitedPoolType lpt : values()) {
```

Comment:
this loop doesn't do anything?

Reply:
@tehdiplomat can you check, i change this PR to reflect the one you suggested (removed the streams and filter).

Reply:
No he's right this loop is later down in line 38. This one is duplicate

Reply:
This loop is counting the size for the be est array

Reply:
Ok so It counts all draftable then set the size of array and fill the array with valid enum values.